pymc/tests/distributions/test_multivariate.py

```diff
@@ -755,7 +755,12 @@ def test_car_logp(self, sparse, size):
 
         # d x d adjacency matrix for a square (d=4) of rook-adjacent sites
         W = np.array(
-            [[0.0, 1.0, 1.0, 0.0], [1.0, 0.0, 0.0, 1.0], [1.0, 0.0, 0.0, 1.0], [0.0, 1.0, 1.0, 0.0]]
+            [
```

Comment:
Your pre-commit is changing unrelated stuff

Reply:
I think this is solved